python_todo/static/css/custom.css

```diff
@@ -0,0 +1,223 @@
+body {
```

Comment:
This all looks good. For bonus points, you should give SCSS try. SCSS allows you to use variables, functions, and nesting, which comes in handy as projects grow. Plus, if you use `bootstrap-sass`, you can actually customize bootstrap's variables to control the colors they use, which is way nicer than overriding styles.

I rarely see people using straight-up CSS on anything non-trivial. And, this will give you an opportunity to practice configuring webpack. Fun!